pkg/minikube/cluster/cluster.go

```diff
@@ -234,6 +251,14 @@ func UpdateCluster(d drivers.Driver) error {
 	return nil
 }
 
+func localkubeURLWasSpecified(config KubernetesConfig) bool {
+	//see if flag is different than default -> it was passed by user
+	if config.KubernetesVersion != constants.DefaultKubernetesVersion {
```

Comment:
More idiomatic to `return config.KubernetesVersion != constants.DefaultKubernetesVersion` ?